include/fmt/format.h

```diff
@@ -4218,12 +4218,18 @@ struct formatter<nested_view<T>> {
 template <typename T>
 struct nested_formatter {
  private:
-  int width_;
+  int width_ = 0;
   detail::fill_t<char> fill_;
   align_t align_ : 4;
   formatter<T> formatter_;
 
  public:
+  // Using initialization list for 'align_' because default member initializers
+  // for bit-fields are C++20
+  constexpr nested_formatter() noexcept(
+      std::is_nothrow_constructible<formatter<T>>::value)
+      : align_(align_t::none) {}
```

Comment:
Let's move initialization of `width_` here to keep all member initialization in one place and remove the comment above.